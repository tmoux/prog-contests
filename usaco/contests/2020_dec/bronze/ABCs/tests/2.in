41 19 22 38 22 19 3
