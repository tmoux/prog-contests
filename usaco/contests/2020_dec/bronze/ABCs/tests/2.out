3 19 19
